### PR TITLE
add 6sense account ads performance

### PR DESCRIPTION
**Pull request by hafizasyraf72**

### CODE REVIEW

File: Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql [hunk 1]
```diff
@@ -1,98 +1,336 @@
...
+  ads.*,
+  airtable_fields._campaign_name,
+  airtable_fields._ad_group,
+  airtable_fields._screenshot,
+  airtable_fields._adtype,
+  airtable_fields._platform,
+  airtable_fields._segment,
+  DATE_TRUNC(_date, MONTH) AS _month_year
+FROM 
+    ads_li ads
```
Comment (izztsfn) [inline Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql:62]: Ensure ads_li ads is in the same line as FROM

File: Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql [hunk 2]
```diff
@@ -1,98 +1,336 @@
...
+),
+
+data_li_6sense AS (
+SELECT 
+  std_name,
+  master.new_industry,
+  ad_name,
+  master.icp_tier,
+  master.customer_segment,
+  CASE WHEN _liextractdate = '6/4/2024' THEN PARSE_DATE('%m/%d/%Y', '5/31/2024')
```
Comment (izztsfn) [inline Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql:248]: Follow the same CASE statement formatting in data_6sense CTE. That one looks correct. Do the same for the rest of the CASE statements, WHEN and ELSE in this CTE

File: Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql [hunk 3]
```diff
@@ -1,98 +1,336 @@
...
+), 
+
+combined_data_impr_li_6sense AS (
+SELECT
+  lagged_li_6sense.*,
+  IFNULL(_impressions - prev_impressions, _impressions) AS change_impressions,
+  IFNULL(_spent - prev_spent, _spent) AS change_spent,
+  IFNULL(_clicks - prev_clicks, _clicks) AS change_clicks
+FROM
+lagged_li_6sense
```
Comment (izztsfn) [inline Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql:312]: Ensure lagged_li_6sense is in the same line as FROM

File: Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql [hunk 4]
```diff
@@ -1,98 +1,336 @@
...
+  CAST(REPLACE(_clicks, '.0', '') AS INTEGER) AS _clicks,
+  CAST(REPLACE(_impressions, ',', '') AS INTEGER) AS _impressions,
+  CASE 
+    WHEN _date LIKE '%/%' THEN PARSE_DATE('%m/%e/%Y', _date)
+    WHEN _date LIKE '%-%' THEN PARSE_DATE('%F', _date) 
+  END AS _date,
+  FROM `x-marketing.blend360_mysql.db_6s_daily_campaign_performance`
+  WHERE _datatype = 'Ad'
+    AND _sdc_deleted_at IS NULL
+  QUALIFY ROW_NUMBER() OVER (PARTITION BY _campaignid, _6senseid, _date ORDER BY CASE WHEN _extractdate LIKE '%/%' THEN PARSE_DATE('%m/%e/%Y', _extractdate) WHEN _extractdate LIKE '%-%' THEN PARSE_DATE('%F', _extractdate) END) = 1
```
Comment (izztsfn) [inline Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql:82]: Too long. Can do something like this instead:

```
QUALIFY ROW_NUMBER() OVER (
  PARTITION BY _campaignid, _6senseid, _date
  ORDER BY
    CASE
      WHEN _extractdate LIKE '%/%' THEN PARSE_DATE('%m/%e/%Y', _extractdate)
      WHEN _extractdate LIKE '%-%' THEN PARSE_DATE('%F', _extractdate)
    END
) = 1
```

File: Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql [hunk 5]
```diff
@@ -1,98 +1,336 @@
...
+  airtable_fields._ad_group,
+  airtable_fields._screenshot,
+  airtable_fields._adtype,
+  airtable_fields._platform,
+  airtable_fields._segment,
+  DATE_TRUNC(_date, MONTH) AS _month_year
+FROM 
+    ads_li ads
+JOIN airtable_fields_li airtable_fields
+    ON ads._adid = airtable_fields._ad_id
```
Comment (izztsfn) [inline Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql:64]: reduce one indentation for ON

**Review by izztsfn — CHANGES_REQUESTED** [review]
Some minor formatting changes to make. Thanks!

File: Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql (hunk 1 above)
Comment (hafizasyraf72) [reply Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql:62]: image

amended

File: Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql (hunk 5 above)
Comment (hafizasyraf72) [reply Clients/Blend360/Normalization Script/Ads/blend360_db_6sense_ad_performance.sql:64]: image

amended

**Review by izztsfn — APPROVED** [review]
Looks good formatting wise! Good job.

**Pull request merged**